GitUI/CommandsDialogs/FormResolveConflicts.Designer.cs

```diff
@@ -273,6 +275,13 @@ private void InitializeComponent()
             this.openWithToolStripMenuItem.Text = "Open With";
             this.openWithToolStripMenuItem.Click += new System.EventHandler(this.openWithToolStripMenuItem_Click);
             // 
+            // openFolderToolStripMenuItem
+            // 
+            this.openFolderToolStripMenuItem.Name = "openFolderToolStripMenuItem";
+            this.openFolderToolStripMenuItem.Size = new System.Drawing.Size(195, 22);
+            this.openFolderToolStripMenuItem.Text = "Open Containing Folder";
```

Comment:
Please align the text with the existing wording, e.g.,
![image](https://user-images.githubusercontent.com/4403806/169633978-ac83de39-d978-4c15-9d16-949b172cbed6.png)

Adding an icon would be a good move too.

We also use the sentence casing.

Reply:
Fixed - have updated the screenshot in the original description